bool InitializeAutoMode
bool Continue
bool Pause
bool EmergencyStop
bool ManualModeOverride
bool CancelLastTask
bool StartTask

